src/Tizen.NUI/src/internal/Application/Application.cs

```diff
@@ -548,6 +598,49 @@ public enum MemoryStatus
         private NUIApplicationAppControlEventCallbackDelegate applicationAppControlEventCallbackDelegate;
         private ApplicationControlSignal appControlSignal;
 
+        // [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
```

Comment:
could you remove unused code?
also there are same unused codes shown below, please remove them also.